modules/platform/nuxeo-platform-web-common/src/main/java/org/nuxeo/ecm/platform/web/common/requestcontroller/service/FilterConfigDescriptor.java

```diff
@@ -113,7 +104,7 @@ public boolean isPrivate() {
     }
 
     public String getCacheTime() {
-        if (cacheTime == null || cacheTime.equals("")) {
+        if (StringUtils.isBlank(cacheTime)) {
             cacheTime = DEFAULT_CACHE_DURATION;
```

Comment:
Can't this be put on the field itself now? I thought your changes allowed that.

Reply:
Indeed, and there's a "trim" attribute too so no need for the check :+1: